extensions/amp-story/show-tooltip.md

```diff
@@ -0,0 +1,59 @@
+---
+$category@: presentation
+formats:
+  - stories
+teaser:
+  - A pop-up that shows up before navigating away from the current AMP story page.
+---
+
+<!--
+Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# show-tooltip
+
+## Usage
+Tooltips allow users a second-click confirmation before navigating to linked content that takes them away from the current AMP story page. `show-tooltip` is an attribute that can be added to any link on an AMP story page.
```

Comment:
I think explaining what this attribute can be used for, which you do in the `Attribute Values` section, should be part of this section. Otherwise it looks like you can disable tooltips entirely